# Tools-for-Bioinformatics
Here are some scripts that I usually use to process and analyze data.    

### Monte_Carlo_method
Monte Carlo test for the coordinates of PCA or PCoA
### Process_data_from_qiita
1. download data from qiita on remote server
2. split qiita data to different samples
### Machine_Learning
Provide two ML scripts which are often used in microbiome analysis —— XGBoost and Randomforest.   
You can run the rscript add "-h" to see the detail help informations.

##### By the way, Parallel-META which I participated in the development is a comprehensive and full-automatic computational toolkit for rapid data mining among metagenomic datasets. Many commonly used methods of bioinformatics analysis are integrated in the software, it is strongly recommended that you use it.

